docs/Policies/App Policies/index.md

```diff
@@ -0,0 +1,247 @@
+
+### preconsented_groups
+
+This field defines additional groups which are assigned to this application, with the difference that the user is expected to have consented to using these permissions separately from the HMI (via a mobile popup, etc.). Any groups defined in this list do not require consent from the user, even if they have a `user_consent_prompt` defined. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise these permissions are simply added to the normal `groups` list.
+
+### nicknames
+
+Defines a list of potential appNames which an application is allowed to register with. If the appName field in the app's `RegisterAppInterface` does not match one of the values in this list, the message will be rejected. If omitted, no restrictions will be applied to appName.
+
+!!! note
+This field is mandatory when defining policies for a cloud application, in which case the first nickname will be used in the application list prior to app registration.
+!!!
+
+### AppHMIType
+
+Defines a list of AppHMITypes which an application is allowed to register with. If omitted, all AppHMITypes will be allowed.
+
+### keep_context
+
+This field is a flag which allows or disallows an application to use soft buttons with the `KEEP_CONTEXT` action. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise this permission is always allowed.
+
+### steal_focus
+
+This field is a flag which allows or disallows an application to use soft buttons with the `STEAL_FOCUS` action. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise this permission is always allowed.
+
+### priority
+
+String value tied to the `notifications_per_minute_by_priority` section of `module_config`. By assigning a priority value from this list, the application is allowed the number of background alerts which is specified by that category (see [notifications_per_minute_by_priority](../policy-fields#notifications_per_minute_by_priority)).
+
+### default_hmi
+
+Defines the default HMI level that the application will be initialized to when it is registered for the first time. Only applies when Core is built with `EXTERNAL_PROPRIETARY` policy mode, otherwise the default HMI level is always `NONE`. 
+
+### RequestType
+
+Defines a list of possible values for `requestType` that can be used in `OnSystemRequest` and `SystemRequest` RPCs when communicating with this app.
+
+### RequestSubType
+
+Defines a list of possible values for `requestSubType` that can be used in `OnSystemRequest` and `SystemRequest` RPCs when communicating with this app.
+
+### memory_kb
+
+Unused, see https://github.com/smartdevicelink/sdl_core/issues/3127.
+
+### heart_beat_timeout_ms
+
+Heartbeat timeout for this application, deprecated in [Protocol Version 4](https://github.com/smartdevicelink/protocol_spec#45-heartbeat).
```

Comment:
It might be helpful to have a date here as well, so people can quickly tell if it was deprecated before/after they started using Core

Reply:
In practice, this has been the case almost since the project was made open source, but I can link to Release 4.0.0, where protocol version 4 was introduced.